var/spack/repos/builtin/packages/dbcsr/package.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
```

Comment:
@tgamblin does it matter whether there's an extra `#` here?

@davydden to make this consistent the license header would be:

```
# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
# Spack Project Developers. See the top-level COPYRIGHT file for details.
# 
# SPDX-License-Identifier: (Apache-2.0 OR MIT)
```

Reply:
i updated the header.